docs/designers-developers/developers/block-api/block-supports.md

```diff
@@ -194,6 +194,45 @@ supports: {
 }
 ```
 
+## duotone
+
+- Type: `boolean`, `string`, `string[]`, or `object`
+- Default value: `false`
+
+This property adds block controls which allow to apply a duotone filter to a block or part of a block. _Important: It doesn't work with dynamic blocks yet._
+
+```js
+supports: {
+    // Declare support for duotone.
+    // This applies the duotone filter to the entire block in both edit and save.
+    duotone: true
```

Comment:
I noticed that in the "theme.json" of  the other PR;  "duotone" was under "color",  while it's not here? Do we expect these two things to apply to separate selectors, should we just make it part of the "color" support flag instead?

Reply:
Now that duotone block support is under color, how important is it to for the API match the other custom colors? i.e. How important is it for there be a separate duotone attribute for the preset slugs separate from the custom duotone under the style attribute? And is it okay if support is only added via theme.json instead of `add_theme_support`?

Reply:
If we don't separate out the slug into its own attribute, should the slug still be used in place of the generated ID when it's a preset?

Related discussion https://github.com/WordPress/gutenberg/pull/26751#discussion_r583462067